telco-core/configuration/reference-crs-kube-compare/required/networking/NMState.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: nmstate.io/v1
```

Comment:
the presence of nmstate resources should be optional and probably should depend on the installation state of the nmstate sub. not all customers will be modifying their networking